task/mongo-tasks.js

```diff
@@ -197,7 +374,43 @@ async function task_1_14(db) {
  *       https://docs.mongodb.com/manual/reference/operator/aggregation/dateFromString/
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    const ordersInMonths = await db.collection('orders').aggregate([
+        {
+            $match: {
+                $expr: {
+                    $eq: [{$year: {$dateFromString: {dateString: "$OrderDate"}}}, 1997]
+                }
+            }
+        },
+        {
+            $project: {
+                month: { "$month": { $dateFromString: {dateString: "$OrderDate"} } }
+            }
+        },
+        {
+            $group: {
+                "_id": "$month", 
+                "orders": { "$sum": 1 }
+            }
+        }
+    ]).toArray();
+
+    const result = {
+        "January": ordersInMonths.filter(e => e._id === 1)[0].orders,
```

Comment:
Looks goon in general, but for the task please try to make all of the operations on DB side.

Reply:
Done